docs/standard/serialization/system-text-json-how-to.md

```diff
@@ -487,16 +487,48 @@ To serialize the properties of the derived type, use one of the following approa
 
 In the preceding example scenario, both approaches cause the `WindSpeed` property to be included in the JSON output:
 
+```json
+{
+  "WindSpeed": 35,
+  "Date": "2019-08-01T00:00:00-07:00",
+  "TemperatureCelsius": 25,
+  "Summary": "Hot"
+}
+```
+
+> [!IMPORTANT]
+> These approaches provide polymorphic serialization only for the root object to be serialized, not for properties of that root object. 
+
+You can get polymorphic serialization for lower-level objects if you define them as type `Object`. For example, suppose your `WeatherForecast` class has a property named `PreviousForecast` that can be defined as type `WeatherForecast` or `Object`:
+
+[!code-csharp[](~/samples/snippets/core/system-text-json/csharp/WeatherForecast.cs?name=SnippetWFWithPrevious)]
```

Comment:
Are we going to add these first before this PR gets merged?

Reply:
Yes, I've started a new PR for this, which will be merged before this one: https://github.com/dotnet/samples/pull/1897